test/array-spec.js

```diff
@@ -37,6 +37,7 @@ describe('Array validation type', function() {
       expect(syncFunction).withArgs(doc).to.throwException(function(ex) {
         expect(ex.forbidden).to.contain('Invalid arrayDoc document');
         expect(ex.forbidden).to.contain('length of item "lengthValidationProp" must not be less than 2');
+        expect(numberOfValidationErrors(ex.forbidden)).to.be(1);
```

Comment:
You'll find that I added this bit of verification to all of the existing unit test cases to ensure that no extra validation errors accidentally slip by unnoticed.